### PR TITLE
Add line breaks option

### PR DESCRIPTION
**Pull request by litzebauer**
Added an option to specify LF or CRLF line breaks

### CODE REVIEW

**Comment by misteroneill** [comment]
Great idea - thanks!

File: index.js [hunk 1]
```diff
@@ -71,6 +71,22 @@ function getPath(fn, cwd) {
...
+ *
+ * @private
+ * @param   {String} lineBreaks
+ * @return  {String}
+ */
+function getLinebreak(lineBreaks) {
+    if (lineBreaks && lineBreaks.toLowerCase() === 'crlf') {
+        return '\r\n';
+    }
+    else {
```
Comment (misteroneill) [inline index.js]: Really minor, but do you think you could change this to the `} else {` style?

File: README.md [hunk 2]
```diff
@@ -56,3 +56,4 @@ The `bannerize` module can be used in your programs. It exports a single functio
 
 - `banner` A banner file location. Defaults to `banner.ejs` in the `cwd`.
 - `cwd` Override the `cwd` for all paths passed to `bannerize`. Relative paths will be relative to `process.cwd()`. Defaults to `process.cwd()`.
+- `lineBreak` Sets the linebreak (CRLF, LF). Defaults to LF.
```
Comment (misteroneill) [inline README.md]: Could we change this to:

```
Sets the linebreak (`'CRLF'`, `'LF'`). Defaults to `'LF'`.
```

**Comment by misteroneill** [comment]
LGTM! Thanks for doing this. I'll get it merged and released when I have some free time later today.

**Comment by litzebauer** [comment]
Awesome!

**Comment by misteroneill** [comment]
v1.1.0 is published to npm 👍

**Pull request merged**